game-core/src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -207,7 +218,7 @@ private void parseMapDetails(final Element root) throws GameParseException {
     data.getRelationshipTracker().setSelfRelations();
     // set default tech attachments (comes after we parse all technologies, parse all attachments, and parse all game
     // options/properties)
-    if (data.getGameLoader() instanceof TripleA) {
+    if (!noValidation) {
```

Comment:
This is what I was alluding to above in terms of readability.  The negative of a negative in this case really means "if validating," right?